src/typhoon_automator/automator.py

```diff
@@ -151,7 +154,9 @@ def add_data_logger_signals(
         :param list[str] signals: Names of streaming signals to be logged
         :raises ValueError: The loaded schematic does not contain one or more of the signal names
         """
-        raise NotImplementedError()
+        if not signals or len(signals) == 0:
```

Comment:
This is correct, but we can tighten it up just a bit:
```suggestion
        if (not signals) or (len(signals) < 1):
```